packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -275,7 +275,7 @@ const withChunking = (config) => {
 const staticFolderCopyPlugin = new CopyPlugin({
     patterns: [
         {
-            from: 'static',
+            from: 'src/static',
```

Comment:
Running `npm run build` and I can see that the static images are there in the build now 👍